pythonforandroid/recipe.py

```diff
@@ -1024,8 +1024,12 @@ def cythonize_file(self, env, build_dir, filename):
             del cyenv['PYTHONPATH']
         if 'PYTHONNOUSERSITE' in cyenv:
             cyenv.pop('PYTHONNOUSERSITE')
-        cython_command = sh.Command(self.ctx.cython)
-        shprint(cython_command, filename, *self.cython_args, _env=cyenv)
+        cython_command = sh.Command(
```

Comment:
technically this isn't a cython command, but a python command, correct?
The cython however comes later once you adds the `-m cython` to it few lines below